app/styles/app.scss

```diff
@@ -52,4 +52,13 @@ body.centered-modal-showing {
 .modal-container {
   max-height: 80vh;
   overflow: auto;
-}
\ No newline at end of file
+}
+
+.ember-modal-overlay {
```

Comment:
Don't modify ember-modal-overlay classes directly, Instead use your own custom classes.
See `custom styles` sections here: http://yapplabs.github.io/ember-modal-dialog/

Reply:
Actually, this what @vdvibhu20 asked me to do! But I'll update!